# DisConST
spatial domain identification


## Overview

Spatial transcriptomics (ST) is a cutting-edge technology that comprehensively characterizes gene expression patterns from a spatial perspective. Spatial domain identification, a pivotal research direction within spatial transcriptomics, serves as a critical reference for in-depth exploration of tissue organization, biological development, and disease studies, among other downstream analyses. In this work, we introduce DisConST, a novel approach that employs an optimization strategy based on zero-inflated negative binomial (ZINB) distribution and graph contrastive learning to capture the most effective representations of ST data. The representations has integrated spatial position information, transcriptome, and cell type proportions. Validation of DisConST across diverse datasets, including normal tissues and organs from various sequencing platforms, diseased tissues, and embryos, consistently yielded higher spatial domain recognition accuracy than existing methods. Meanwhile, these experiments also validated the value of DisConST in tissue organization, disease research, and biological development.

## Software dependencies

